### PR TITLE
[ISSUE #1607] support call server with streaming in grpc.

### PR DESCRIPTION
**Pull request by midnight2104**
// Describe your PR here; eg. Fixes #issueNo
[ISSUE #1607] support call server with streaming in grpc.

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/projects/soul/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/resources/application.yml [hunk 1]
```diff
@@ -88,25 +87,9 @@ shenyu:
...
-      - /configs/**
-      - /shenyu-client/**
-      - /error
-      - /actuator/health
-      - /swagger-ui.html
-      - /webjars/**
-      - /swagger-resources/**
-      - /v2/api-docs
-      - /csrf
+    white-list: /,/favicon.*,/static/**,/index**,/plugin,/platform/**,/websocket,/configs/**,/shenyu-client/**,/error,/actuator/health
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/application.yml:90]: not modify this

Comment (midnight2104) [reply shenyu-admin/src/main/resources/application.yml:90]: I'm not modify this, i just merge master.

Comment (midnight2104) [reply shenyu-admin/src/main/resources/application.yml:90]: I made an error while merging,

File: shenyu-admin/src/main/resources/application.yml [hunk 2]
```diff
@@ -72,8 +72,8 @@ shenyu:
   #        namespace:
   #        accessKey:
   #        secretKey:
-#    etcd:
-#      url: http://localhost:2379
+  #    etcd:
+  #      url: http://localhost:2379
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/application.yml:76]: not commit this file

**Review by yu199195 — APPROVED** [review]

**Pull request merged**